flink-core/src/main/java/org/apache/flink/core/fs/RefCountedBufferingFileStream.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.flink.fs.s3.common.utils;
+package org.apache.flink.core.fs;
```

Comment:
Need to update `RefCountedBufferingFileStream.sync`

Reply:
OK